README.md

```diff
@@ -625,6 +625,7 @@ robot.
 * [ORB_SLAM3](https://github.com/UZ-SLAMLab/ORB_SLAM3) - ORB-SLAM3: An Accurate Open-Source Library for Visual, Visual-Inertial and Multi-Map SLAM
 * [Atlas](https://github.com/magicleap/Atlas) - End-to-End 3D Scene Reconstruction from Posed Images
 * [vilib](https://github.com/uzh-rpg/vilib) - This library focuses on the front-end of VIO pipelines with CUDA
+* [iris_lama](https://github.com/iris-ua/iris_lama) - LaMa is a C++11 software library for robotic localization and mapping
```

Comment:
Should be put under lidar ..

Reply:
Thanks for the hint.  :+1: Changed that.